ownCloudAppShared/UIKit Extension/LAContext+Extension.swift

```diff
@@ -17,18 +17,38 @@
  */
 
 import LocalAuthentication
+import UIKit
 
 extension LAContext {
+	public func supportedBiometricsAuthenticationName() -> String? {
+		if  canEvaluatePolicy(.deviceOwnerAuthenticationWithBiometrics, error: nil) {
+			switch self.biometryType {
+				case .faceID : return "Face ID".localized
+				case .touchID: return "Touch ID".localized
+				case .none: return nil
+				@unknown default: return nil
+			}
+		}
+		return nil
+	}
 
-    public func supportedBiometricsAuthenticationName() -> String? {
-
-        if  canEvaluatePolicy(.deviceOwnerAuthenticationWithBiometrics, error: nil) {
-            switch self.biometryType {
-            case .faceID : return "Face ID".localized
-            case .touchID: return "Touch ID".localized
-            case .none: return nil
-            }
-        }
-        return nil
-    }
+	public func biometricsAuthenticationImage() -> UIImage? {
+		if  canEvaluatePolicy(.deviceOwnerAuthenticationWithBiometrics, error: nil) {
+			switch self.biometryType {
+				case .faceID : if #available(iOSApplicationExtension 13.0, *) {
+					return UIImage(systemName: "faceid")
+				} else {
+					return UIImage(named: "biometrical-faceid")
+				}
+				case .touchID: if #available(iOSApplicationExtension 13.0, *) {
+					return UIImage(systemName: "touchid")
```

Comment:
iOS 13 switch is no longer needed

Reply:
I removed the pre-iOS 13 code here as well. Thanks!